Sources/ProjectDescription/Project.swift

```diff
@@ -6,19 +6,16 @@ public class Project: Codable {
     public let name: String
     public let targets: [Target]
     public let settings: Settings?
-
-    public enum CodingKeys: String, CodingKey {
-        case name
-        case targets
-        case settings
-    }
+    public let additionalFiles: [WorkspaceElement]
```

Comment:
Is this still a workspace element?

Reply:
If we'd like to reuse the same type I'd change the name.

Reply:
I'd stick to the Xcode project's convention, [FileElement](https://github.com/tuist/xcodeproj/blob/master/Sources/xcodeproj/Objects/Files/PBXFileElement.swift#L5)